jmclient/jmclient/wallet_utils.py

```diff
@@ -997,6 +997,9 @@ def wallet_showseed(wallet):
 def wallet_importprivkey(wallet, mixdepth, key_type):
     jmprint("WARNING: This imported key will not be recoverable with your 12 "
           "word mnemonic phrase. Make sure you have backups.", "warning")
+    jmprint("WARNING: Make sure that the type of the public address previously "
+          "derived from this private key matches the wallet type you are "
+          "currently using.")
```

Comment:
You could have added "warning" as a second argument to the fn call here but it's trivial. Fix it later if you want.